3D Chess/Assets/Scripts/GameState.cs

```diff
@@ -6,15 +7,31 @@ public class GameState : MonoBehaviour
     [SerializeField] private PieceSpawner pieceSpawner = default;
     [SerializeField] private GameObject pieces = default;
 
+    private Realm realm = default;
+    private IQueryable<PieceEntity> pieceEntities = default;
```

Comment:
These don't get populated by Unity, right? So they don't need to be assigned to `default` or do they?

Reply:
No, you're totally right. I've just gotten used to having `default` everywhere because of Unity that I redundantly added it there too. Removed with 0ff6bbb.